spec/acceptance/sources_spec.rb

```diff
@@ -215,6 +249,55 @@
     end
   end
 
+  context 'MODULES-4418 - Change Existing Bypass Proxy Setting' do
+
+    before(:all) do
+      backup_config
+    end
+    
+    after(:all) do
+      reset_config
+    end
+
+    windows_agents.each do | agent |
+      it 'Should apply a manifest to set bypass_proxy' do
+        chocolatey_src = <<-PP
+          chocolateysource {'chocolatey':
+            ensure   => present,
+            location => 'https://chocolatey.org/api/v2',
+            bypass_proxy => true,
+          }
+        PP
+
+        execute_manifest_on(agent, chocolatey_src, :catch_failures => true)
+      end
+
+      it 'Should verify the setup was added' do
+        on(agent, config_content_command) do | result |
+          assert_match(/true/, get_xml_value("//sources/source[@id='chocolatey']/@bypassProxy", result.output).to_s, 'Bypass Proxy setup did not match')
+        end
+      end
+
+      it 'Should apply a manifest to change the priority' do
```

Comment:
*Should apply manifest to set bypass_proxy to false

Reply:
Fixed.